style.css

```diff
@@ -0,0 +1,145 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400&display=swap');
+
+:root {
+  --global-border: 1px solid black;
+}
+
+body {
+  /* styling the body */
+  background: #e5e5e5;
+
+  /* centering the container */
+  display: flex;
+  justify-content: center;
+}
+
+#background-mobile {
+  height: 718px;
+  width: 375px;
+  background-color: #fff;
+  overflow: hidden;
+  border-radius: 0 0 0 70px;
+}
+
+#status-bar {
+  width: 100%;
+  height: 44px;
+  background: url(/Pictures/Status-Bar.png);
+}
+
+header {
+  height: 48px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-align: center;
+  flex-direction: row;
+}
+
+header h2 {
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  margin-left: 24px;
+}
+
+header #menu-bar {
+  background: url(/Pictures/Union.svg);
```

Comment:
- [ ] [OPTIONAL] - I'd urge you to always put a dot in front of every file path as different operating systems resolve file paths differently. This in my opinion should be changed to `background: url(./Pictures/Union.svg);` This might be a small change but one which will help you from a lot of problems with future reviewers not seeing your images.